components/Dialogs/docs/theming.md

```diff
@@ -3,7 +3,15 @@
 You can theme an MDCDialog to match the Material Design Dialog using your app's schemes in the DialogThemer
 extension.
 
-You must first add the DialogThemer extension to your project:
+Make sure the Dialog's Theming extension is added to your project.
+
+If you're using the new theming extentions and schemes (MDCContainerScheme), add:
+
+```bash
+pod 'MaterialComponents/Dialogs+Theming'
```

Comment:
This doesn't work. For Beta components we'll have to give instructions to add them to a project.

I just attempted the same Podfile config you have here and it fails.

```
rsmoore-macbookpro1:test-pods rsmoore$ pod install
Analyzing dependencies
[!] CocoaPods could not find compatible versions for pod "MaterialComponents/Dialogs+Theming":
  In Podfile:
    MaterialComponents/Dialogs+Theming

None of your spec sources contain a spec satisfying the dependency: `MaterialComponents/Dialogs+Theming`.

You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.

Note: as of CocoaPods 1.0, `pod repo update` does not happen on `pod install` by default.

[!] Automatically assigning platform `ios` with version `12.0` on target `test-pods` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.
rsmoore-macbookpro1:test-pods rsmoore$ cat Podfile
# Uncomment the next line to define a global platform for your project
# platform :ios, '9.0'

target 'test-pods' do
  # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
  # use_frameworks!

  # Pods for test-pods
  pod 'MaterialComponents/Dialogs+Theming'

  target 'test-podsTests' do
    inherit! :search_paths
    # Pods for testing
  end

  target 'test-podsUITests' do
    inherit! :search_paths
    # Pods for testing
  end

end
```